test/runtest.py

```diff
@@ -758,9 +758,6 @@ def testGetMultiple(self):
         results = ray.get([object_ids[i] for i in indices])
         self.assertEqual(results, indices)
 
-    @unittest.skipIf(
-        os.environ.get("RAY_USE_XRAY") == "1",
-        "This test does not work with xray yet.")
```

Comment:
I think there are other tests that can be added back in, e.g., `testMultipleWaitsAndGets` and `testWait` in `stress_tests.py`.